staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -234,11 +234,12 @@ func (cfgCtlr *configController) updateObservations() {
 	}
 }
 
-// used from the unit tests only.
-func (cfgCtlr *configController) getPriorityLevelState(plName string) *priorityLevelState {
+// intended to be used by the unit tests only.
```

Comment:
Can we move this method to `controller_test.go` ?

Reply:
@MikeSpreitzer  done.